multi_import/object_cache.py

```diff
@@ -32,37 +32,73 @@ def __init__(self, lookup_fields):
     def _default_factory(self):
         return defaultdict(set)
 
-    def _add_lookup_value(self, field, value, instance):
+    def is_multiple_fields(self, fields):
+        return (
+            hasattr(fields, '__iter__')
+            and not isinstance(fields, (str, bytes))
```

Comment:
Actually, ignore my last comment - I misread/mis-understood for a moment there.

Yeah, I could change this check to explicitly look for tuple or list. I don't see any harm in that.